frontend/src/libs/components/app/app.tsx

```diff
@@ -29,6 +30,8 @@ const App: React.FC = () => {
     return <Loader />;
   }
 
+  const image = '../../../../public/images/meditation-image.svg';
```

Comment:
const image = 'images/meditation-image.svg';

Reply:
Oh, it is a temporary const, I will delete it
Thanks, for your attention